### PR TITLE
feat: create a new build command

### PR DESCRIPTION
**Pull request by Gillespie59**
Fixes https://github.com/googlecodelabs/tools/issues/78.

### CODE REVIEW

**Comment by samtstern** [comment]
@x1ddos I believe this was in reference to https://github.com/googlecodelabs/tools/issues/78

Not sure about the `install` name but I think it's a good idea to have a command that gets the codelab ready for serving but without dictating/starting the server.  I could see this being use for CI or static builds.

**Comment by x1ddos** [comment]
Ah ok. That makes more sense. How about actually using the proposed `build` command? Sounds closer to that it's supposed to do.

Either way, one thing missing from this change is a help section in main.

File: claat/serve.go [hunk 1]
```diff
@@ -15,41 +15,15 @@
...
-	}
-	// Go get the dependencies.
-	if err := fetchRepo(depsDir, "google-codelab-elements", "googlecodelabs/codelab-components#2.0.2"); err != nil {
-		fatalf(err.Error())
-	}
-	if err := writeFile(filepath.Join("elements", "codelab.html"), codelabElem); err != nil {
-		fatalf(err.Error())
-	}
-
+	cmdInstall()
```
Comment (x1ddos) [inline claat/serve.go]: This is now `cmdBuild()`.

File: claat/main.go [hunk 2]
```diff
@@ -180,6 +181,10 @@ When writing to a directory, existing files will be overwritten.
 
 The program exits with non-zero code if at least one src could not be exported.
 
+## Build command
+
+Install all the dependencies needed by the codelab.
```
Comment (x1ddos) [inline claat/main.go]: Let's add something like:

> Install all the ... unless already installed.
> This is done automatically by the serve command.
> To clean up and rebuild all the dependencies, remove bower_components
> directory and run the build command again.

File: claat/main.go [hunk 3]
```diff
@@ -188,6 +193,8 @@ Clicking on a directory representing an exported codelab will load
 all the required dependencies and render the generated codelab as
 it would appear in production.
 
+The serve command run first the build command.
```
Comment (x1ddos) [inline claat/main.go]: "The serve command always runs the build command first." or something like that. I'm not a native. @samtstern please correct.

Comment (samtstern) [reply claat/main.go]: @x1ddos yeah your suggestion LGTM

**Pull request merged**